src/components/header/server-utils.ts

```
@@ -0,0 +1,110 @@
+        
+      case CollectionErrorType.PERMISSION_ERROR:
+        console.error("Erreur de permissions - vérifier la configuration Astro Content");
+        break;
+        
+      case CollectionErrorType.CONTENT_ERROR:
+        console.warn("Erreur de structure de contenu - vérifier le schema des articles");
+        break;
+        
+      case CollectionErrorType.UNKNOWN_ERROR:
+      default:
+        console.error("Erreur inconnue lors de l'accès aux articles");
```

Biome (1.9.4)

[error] Useless case clause.

because the default clause is present:

Unsafe fix: Remove the useless case.


(lint/complexity/noUselessSwitchCase)